src/main/kotlin/com/example/delivery/controller/UserController.kt

```diff
@@ -28,6 +30,11 @@ class UserController(private val userService: UserService) {
         return userService.getById(id).toDTO()
     }
 
+    @PatchMapping("/{id}")
```

Comment:
This is multipurpose API which I see is used to change Order lifecycle and Order props like ship details. I think better to have separate API for pure update details and change lifecycle. Then you will not need to have such Patch at all since PUT is more suitable for update. Patch is more complex to maintain